### PR TITLE
allow avatar url to be longer

### PR DESCRIPTION
**Pull request by porterjamesj**

### CODE REVIEW

File: db/migrate/20201106150006_increase_max_length_of_avatar_url.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class IncreaseMaxLengthOfAvatarUrl < ActiveRecord::Migration[5.2]
+  def change
+    change_column :users, :avatar_url, :string, :limit => nil
```
Comment (davidbalbert) [inline db/migrate/20201106150006_increase_max_length_of_avatar_url.rb:3]: You're using `change` but `change_column` doesn't know how to go backwards. Can you change it to `up` and `down`

Comment (porterjamesj) [reply db/migrate/20201106150006_increase_max_length_of_avatar_url.rb:3]: ahh, sure, and I guess I would put `change_column :users, :avatar_url, :string, :limit => 255` in the `down`?

Comment (davidbalbert) [reply db/migrate/20201106150006_increase_max_length_of_avatar_url.rb:3]: Yup

Comment (porterjamesj) [reply db/migrate/20201106150006_increase_max_length_of_avatar_url.rb:3]: OK, just tested and that doesn't work if there are any values longer than 255 in the table, so it's not very practically useful, but I can include it just for completenesses sake unless you have a better idea

Comment (davidbalbert) [reply db/migrate/20201106150006_increase_max_length_of_avatar_url.rb:3]: Eh, whatever. I don't think it matters. Leave it as is

**Pull request merged**